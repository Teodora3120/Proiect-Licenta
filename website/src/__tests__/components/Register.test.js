import React from 'react'
import { shallow } from 'enzyme'
import Register from '../../views/examples/Register'
it('should test Register component', () => {
  const wrapper = shallow(<Register />)
  expect(wrapper).toMatchSnapshot()
})
