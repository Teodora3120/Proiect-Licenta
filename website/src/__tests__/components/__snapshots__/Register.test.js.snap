// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Register component 1`] = `
<Fragment>
  <Col
    lg="6"
    md="8"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Card
      className="bg-secondary shadow border-0"
      tag="div"
    >
      <r
        className={null}
        initialStep={1}
        instance={[Function]}
        isHashEnabled={false}
        isLazyMount={false}
        nav={null}
        onStepChange={[Function]}
        transitions={Object {}}
      >
        <FirstStep
          handleType={[Function]}
        />
        <SecondStep
          accountType=""
          creating={false}
          error={null}
          handleRegister={[Function]}
          setEmail={[Function]}
          setFirstName={[Function]}
          setLastName={[Function]}
          setPassword={[Function]}
          setRepeatPassword={[Function]}
          setShowPassword={[Function]}
          showPassword={false}
          stepWizardRef={null}
        />
        <ThirdStep
          email=""
          firstName=""
          lastName=""
          password=""
        />
      </r>
    </Card>
  </Col>
</Fragment>
`;
